[julia-python-homework]/3.conditional_logic/3.travel_itinerary_planner.py

```diff
@@ -0,0 +1,17 @@
+"""Plan your future holiday
+# Input: The user enters three cities they plan to visit.
+# Output: The program outputs a travel itinerary in the order entered.
+# Input:
+# >> London
+# >> Paris
+# >> Rome
+# Output:
+# Your travel itinerary: London -> Paris -> Rome"""
+
+city_1 = input("First city: ")
+city_2 = input("Second city: ")
+city_3 = input("Thrid city: ")
+print("Your travel itinerary :", city_1, " -> ", city_2, " -> ", city_3)
+
+
+## Is this how it should be?
```

Comment:
This is correct, but each output should have been on the separate line. Consider adding `/n` as a `sep` param